sources/osgShader/node/shadows.js

```diff
@@ -28,9 +28,13 @@ define( [
         defines: function () {
             return this._shadow.getDefines();
         },
-
+        extensions: function () {
```

Comment:
we should be consistent and use get when it does get, so extensions should be getExtensions, define getDefines.
If you can adjust, for the new field and make it backward compatible for other with a warning